packages/app/src/compositions/useAuthentication.ts

```diff
@@ -0,0 +1,23 @@
+import { ref, computed } from 'vue';
+import { User } from '@project-leek/commons';
+
+import feathers from '../lib/feathers';
+
+export const user = ref<User | null>(null);
+
+// !!user.value to cast the user variable to a boolean
```

Comment:
I would not put that here, as you would know what this code does as soon as you know the (double)negation operator.
```suggestion
```